lib/cli.js

```diff
@@ -60,7 +62,10 @@ if (program.filelist) {
   rl.on("close", () => {
     clearBuffer();
     try {
-      print(cpa);
+      const duplicates = print(cpa);
```

Comment:
Like I told you `findOptimalDuplicates` function in `cpa` object always returns a `[]` if I call it from `generateHTMLReport`(calling it second time). You need to reset the memory in that function so I can use it correctly.